test/integration/garbagecollector/garbage_collector_test.go

```diff
@@ -534,16 +531,43 @@ func TestStressingCascadingDeletion(t *testing.T) {
 	wg.Add(collections * 5)
 	rcUIDs := make(chan types.UID, collections*5)
 	for i := 0; i < collections; i++ {
+		// use a channel to hand off the error
 		// rc is created with empty finalizers, deleted with nil delete options, pods will remain.
-		go setupRCsPods(t, gc, clientSet, "collection1-"+strconv.Itoa(i), ns.Name, []string{}, metav1.DeleteOptions{}, &wg, rcUIDs)
+		errs := make(chan error, 1)
+		go setupRCsPods(t, gc, clientSet, "collection1-"+strconv.Itoa(i), ns.Name, []string{}, metav1.DeleteOptions{}, &wg, rcUIDs, errs)
+		err := <-errs
```

Comment:
blocking on receiving the error means these all run serially, which defeats the purpose of the test. it would be simpler to change the Fatal to an Error

Reply:
Will do.